openamp/contributing.rst

```diff
@@ -4,38 +4,10 @@
 Contributing to the OpenAMP Project
 ===================================
 
-Release Cycle
--------------
-- Release branch (feature freeze) cut 1 month before release target
-- Maintainence releases are left open-ended for now
+You are encouraged to contribute to the OpenAMP project to help create a successful solution through a vibrant and engaged community.
 
-Roadmap discussion and publication
-----------------------------------
-- Feature freeze period of a release used for roadmap discussions for next release
-- Contributors propose features posted and discussed on `mailing list <https://lists.openampproject.org/mailman3/lists>`_
-- Maintainer collects accepted proposals
-- Maintainer posts list of development tasks, owners, at open of release cycle
+Contribution process and guidelines are provided in the project's `Governance Page <https://www.openampproject.org/governance/>`_.
 
-Patch process
--------------
-- Pull request on github for review and merging
-   - Against `OpenAMP main branch <https://github.com/OpenAMP/open-amp/tree/main>`_
-   - Against `libmetal main branch <https://github.com/OpenAMP/libmetal/tree/main>`_
-- Maintainer ensures a minimum of 1 week review window prior to merge
+You can contribute, through pull requests, directly to the main `OpenAMP libraries <https://github.com/OpenAMP>`_ or propose your platform as a reference platform.

```

Comment:
To add:

Any specific contribution requirements will be given in the "How to contribute" section of the repository's README.